packages/docs/src/locales/en/en.json

```diff
@@ -1138,11 +1138,16 @@
         "clearable": "Adds a button to reset the input field value",
         "bordered": "Applies underscore",
         "focused": "Applies focus style",
-        "canBeFocused": "If false, then input can not be focused using mouse or keyboard."
+        "canBeFocused": "If false, then input can not be focused using mouse or keyboard",
+        "icon": "Sets an icon. The default icon position is on the right",
+        "iconLeftPosition": "Sets the icon position to the left"
```

Comment:
Can it be `leftIcon`, same as we have in Button `rightIcon`?

Reply:
Fixed it